data/coalition/coalition missions.txt

```diff
@@ -3111,3 +3111,455 @@ mission "Coalition: Outskirts Gauger Ready"
 			`	While the Arachi work on the outfit, she explains that they used the probe's algorithms to have the scanners excel at finding basic energy signatures, such as heat or energy levels of other ships. "Made to check for complex machinery like outfits, or recognize specific cargo types, it was not, but still find uses for its current form, I'm sure you will."`
 			`	When everything's ready, you two head back inside your ship, and the elevator brings you back up. You fly back to the main landing area, and she pulls close to you just before you open your hatch. "Now, still... experimental, the outfit is, and provide the Heliarchs with a potentially faulty product, we do not wish. So, naturally, not yet registered it with them, we have. Provide you with more copies, we cannot, so careful not to lose this one, you should be." She thanks you again for helping out, and leaves for her workplace.`
 				decline
+
+
+
+conversation "cultural data for coalition group"
+	branch "return"
+		has "Fangvin: Struggling Author - Human Media 1"
+	`Walking through the spaceport, a group of Coalition citizens are on a table speaking in their native languages. They see you. The group then rise from their seats and begin approaching you.`
+	choice
+		`	(Walk at a slightly faster pace.)`
+			goto "walk faster"
+		`	"Excuse me, what do you all want?"`
+			goto "mission intro"
+	label "walk faster"
+	`	You slightly increase your pace, but the group does too. They eventually catch up to you. You turn to face them, as they begin speaking to you.`
+	label "mission intro"
+	`	"Greetings, fellow human. A peaceful group, we are. Curious about your race, we are, so for you, a mission we have.", says one of the Coalition citizens, using a translation box.`
+	choice
+		`	"What is the mission about?`
+	`	"A small mission, it is. Bring us human books, we want you to. Pay you <payment> and be very grateful, we would, if accept this mission, you will.", explains the citizen.`
+	branch "you already have everything"
+		has "human cultural data"
+	choice
+		`	"Sorry, I can't help you."`
+			decline
+		`	"What should I be worried about?"`
+		`	"Is this going to get me in trouble?"`
+			goto "consequences"
+		`	"How much will you pay me?"`
+			goto "ask about pay"
+	label "consequences"
+	`	"The Heliarchs, review everything they must. Heavy-handed, the censorship is. Some books from your space, to read them we want to. Because to break free from Heliarch censorship, we wish. And if forbidden knowledge, on your ship they find, heavy-handed the punishment will be, so avoid scans, you must."`
+	choice
+		`	"I'm not willing to risk it. Sorry."`
+			decline
+		`	"I'm in. What do you want me to collect?"`
+	label "what do they want"
+	`	"Anything. A translation device we have, so the language it is in, of no importance it is. As for the topic, a book about oppressing dictatorships or tyranny, we want. Histories of humanity's resistance movements, we seek."`
```

Comment:
Probably should be ` Everything` either instead or as well. Also, if the mission sends us to Alexandria to get all of human cultural knowledge, anyway, they probably shouldn't be asking for "a book" as the topic at all.

Have them directly ask for more detailed human historic/cultural knowledge than they otherwise have so it makes more sense to immediately bring them all of it. They can still express further interest specifically these same subjects.